### ISSUE TITLE
Backwards button

### ISSUE THREAD

**Issue by meirf** [benbjohnson/thesecretlivesofdata#10]
I think it would add educational value if there wan an option to go backwards step-by-step.
To the left of the continue button, perhaps.

**Comment by chmac** [comment]
Agreed, I sometimes pressed the right key too quickly and skipped a step, but couldn't find a way to go back.

**Comment by retnuh** [comment]
I agree also, there were a few times I would have liked to watch one of the steps again.

**Comment by rushiagr** [comment]
+1. Sometimes we need to go through the learnt things in last section again, just to reinforce them a little bit more..

**Comment by coreyjjohnson** [comment]
Me too. I really like the presentation, but I did click the "next" button accidentally. In order to go back I had to wait for all the animations and that gets frustrating fast.
Thanks for the education on raft! I really appreciate it.

**Comment by desrod** [comment]
There is a menu on the upper-right to skip to the important sections, where you can play forward from there. 

<img width="265" height="197" alt="Image" src="https://github.com/user-attachments/assets/575f9fe7-2f49-47ed-9592-f295536dd2d1" />
